components/Reviews/index.jsx

```diff
@@ -1,42 +1,44 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
When a `Reviews` page is reloaded, the `Header` and the `List` don't show data

Reply:
Done